embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/metadata/EmbraceMetadataService.kt

```diff
@@ -722,6 +722,8 @@ internal class EmbraceMetadataService private constructor(
             return defaultBundleId
         }
 
+        fun isEmulator(): Boolean = MetadataUtils.isEmulator()
```

Comment:
Not a big fan of sticking this here, but I'll convert `MetadatUtil` to kotlin soon and just call it from there as I can make that method internal rather than package private.